### PR TITLE
feat(ld-icon): add icons

### PR DESCRIPTION
**Pull request by borisdiakur**
# Description

This PR adds new icons and updates existing ones. Some icon names have been updated. Previous names still work.

## Type of change

- [x] Feature

## Is it a breaking change?

- [x] No

# How Has This Been Tested?

- [x] tested manually

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [x] New and existing tests pass locally with my changes

### CODE REVIEW

File: src/liquid/components/ld-icon/assets/remove-icon.svg
Comment (renet) [inline src/liquid/components/ld-icon/assets/remove-icon.svg:1]: This is probably a duplicate that should be removed. (If this exists in Figma as well, maybe remove it there, too.)

File: src/liquid/components/ld-icon/assets/add-icon.svg
Comment (renet) [inline src/liquid/components/ld-icon/assets/add-icon.svg:1]: This should be renamed to `add.svg` (and changed accordingly in Figma).

File: src/liquid/components/ld-icon/readme.md [hunk 1]
```diff
@@ -288,10 +288,14 @@ Right-click an icon below to download its SVG file. To download all icons at onc
 </docs-icon-group>
 
 <docs-icon-group name="Modifiers">
-  <docs-icon identifier="add" name="Add"></docs-icon>
+  <docs-icon identifier="add-icon" name="Add Icon"></docs-icon>
+  <docs-icon identifier="remove-icon" name="Remove Icon"></docs-icon>
```
Comment (renet) [inline src/liquid/components/ld-icon/readme.md:292]:
```suggestion
  <docs-icon identifier="add" name="Add"></docs-icon>
  <docs-icon identifier="remove" name="Remove"></docs-icon>
```

**Review by renet — CHANGES_REQUESTED** [review]

File: src/liquid/components/ld-icon/assets/remove-icon.svg
Comment (borisdiakur) [reply src/liquid/components/ld-icon/assets/remove-icon.svg:1]: We previously had "add" which now has been replaced by "add-icon". They are different: "add" is an icon that is required in Figma, but doesn't work with currentColor while "add-icon" works with currentColor. It's a bit complicated. For consistency and backwards-compatibility reasons we now have both. The Liquid Oxygen docs show only the new icons. The old ones still work.

File: src/liquid/components/ld-icon/assets/add-icon.svg
Comment (borisdiakur) [reply src/liquid/components/ld-icon/assets/add-icon.svg:1]: See my comment above.

**Review by renet — APPROVED** [review]

**Pull request merged**